XMonad/Layout/LayoutBuilder.hs

```diff
@@ -58,10 +58,23 @@ module XMonad.Layout.LayoutBuilder (
 ) where
 
 --------------------------------------------------------------------------------
-import Control.Applicative ((<|>))
-import Control.Monad (foldM)
-import Data.Maybe
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
```

Comment:
what's this? :-D

Reply:
Oh ^^'  I think that was `sed` not _quite_ doing what I wanted it to